src/vaults.lua

```diff
@@ -53,8 +55,18 @@ end
 --- @param currentTimestamp number The current timestamp
 --- @param vaultId string The vault id
 --- @param allowUnsafeAddresses boolean|nil Whether to allow unsafe addresses, since this results in funds eventually being sent to an invalid address
---- @return Vault The created vault
-function vaults.vaultedTransfer(from, recipient, qty, lockLengthMs, currentTimestamp, vaultId, allowUnsafeAddresses)
+--- @param revokable boolean|nil Whether the vault is revokable
```

Comment:
Should indicate what the default value is.

Reply:
[2ca8dce](https://github.com/ar-io/ar-io-network-process/pull/334/commits/2ca8dcea8aef1b18d91e3aebbc920f5645474085)